engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/ConsumerGroupDeterminer.scala

```diff
@@ -1,17 +1,18 @@
 package pl.touk.nussknacker.engine.kafka
 
+import pl.touk.nussknacker.engine.api.process.ProcessName
 import pl.touk.nussknacker.engine.flink.api.process.FlinkCustomNodeContext
 
 class ConsumerGroupDeterminer(consumerGroupNamingStrategy: ConsumerGroupNamingStrategy.Value) {
 
   def consumerGroup(nodeContext: FlinkCustomNodeContext): String = {
-    consumerGroup(nodeContext.metaData.id, nodeContext.nodeId)
+    consumerGroup(nodeContext.metaData.name, nodeContext.nodeId)
   }
 
-  def consumerGroup(processId: String, nodeId: String): String = {
+  def consumerGroup(processName: ProcessName, nodeId: String): String = {
```

Comment:
In fact, we should return here ProcessGroupName VO :) But it's not part of your change.. But can you leave here some TODO?

Reply:
Why to introduce VO for such a local think? VO are useful when you have some type that it is commonly used in many places of application. What is `ProcessGroup`? I don't know such a domain concept in our application. Where can I find it in our documentation? It is just a consumer group, the concept from kafka consumers.

Reply:
Ok, it has sens